modules/nw-networkpolicy-about.adoc

```diff
@@ -150,11 +150,15 @@ spec:
 ----
 <1> `policy-group.network.openshift.io/ingress:""` label supports both OpenShift-SDN and OVN-Kubernetes.
 
-
 [id="nw-networkpolicy-allow-from-hostnetwork_{context}"]
-== Using the allow-from-hostnetwork network policy
+== Using the `allow-from-hostnetwork` network policy
+
+Add the following `allow-from-hostnetwork` `NetworkPolicy` object to direct traffic from the host network pods.
 
-Add the following `allow-from-hostnetwork` `NetworkPolicy` object to direct traffic from the host network pods:
+[IMPORTANT]
+====
+If you enabled `HostNetwork` endpoint publishing strategy for your cluster, pods in the `allow-from-host-network` namespace of the host node cannot communicate with pods from a different node in the same namespace.
```

Comment:
On the clusters that have endpoint publishing strategy 'LoadBalancer' the host networked pod on a node cannot communicate with pod on different  node in the same namespace.

Reply:
:heavy_check_mark: